### ISSUE TITLE
Help documentation for Rocket Chat users

### ISSUE THREAD

**Issue by s-duffey** [RocketChat/docs-old#989]
For our instance of Rocket.Chat within the Government of Canada (called GCmessage) we wrote help documentation for our users with more information on how to use the features of the platform.

Since Rocket.Chat's documentation seems to focus more on creating new servers, rather than how to actually use Rocket.Chat features, you may want to look at our help documentation and copy or incorporate some of the content we have written into Rocket.Chat's support.

Our documentation for GCmessage (our version of Rocket Chat) can be found here:

https://gccollab.gctools-outilsgc.ca/en/support/solutions/folders/2100031626/page/1?url_locale=en

**Comment by s-duffey** [comment]
See related: https://github.com/RocketChat/Rocket.Chat/issues/12631

**Comment by shubhangibharti** [comment]
Hi maintainers,

I am a GSoD applicant, have read your ideas list and working on closing tickets present in the list.

I have participated in multiple SoC/WoC and have been involved w/ open source. I have worked with Rest APIs, mobile and software development for chat applications and the project of Application Programming Interfaces (APIs) Documentation seems like a right fit for me.

Can this issue be assigned to me?
